### DISCUSSION TITLE
BlueSCSI2 adding Wifi, do i need a newer version of the BlueSCSI2 board?

### DISCUSSION THREAD

**Discussion by dburr** [BlueSCSI/BlueSCSI-v2#94] [Help] [answered]
I purchased the BlueSCSI2 shortly after it was released.  The board is marked "V2 - 2022.12a" I now see that WiFi functionality is available. Do I need a newer version of the BlueSCSI2 board, or is it just enough to replace my Pi Pico with a Pico W?

**Comment by erichelgeson** [comment, marked as answer]
No, there are no hardware requirements for the board, only the type of Pico. I'd caution you though, it's no small task to desolder and resolder a pico.
